### PR TITLE
use disputeId and arbitratorAddress instead of arbitrableAddress as key

### PR DESCRIPTION
**Pull request by satello**
- use disputeId and arbitratorAddress when looking up disputes
- add indexes

### CODE REVIEW

**Comment by satello** [comment]
closes #9

File: routes/index.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ router.post(
  *   }
  */
 router.post(
-  '/:address/disputes/:disputeHash',
+  '/:address/disputes/:arbitratorAddress/:disputeId',
```
Comment (n1c01a5) [inline routes/index.js]: it's maybe more REST : `/:address/arbitrator/:arbitratorAddress/disputes/:disputeId`

File: routes/index.js [hunk 2]
```diff
@@ -163,12 +163,12 @@ router.post(
  *   }
  */
 router.post(
-  '/disputes/:disputeHash',
+  '/disputes/:arbitratorAddress/:disputeId',
```
Comment (n1c01a5) [inline routes/index.js]: idem `arbitrator/:arbitratorAddress/disputes/:disputeId`

File: routes/index.js [hunk 3]
```diff
@@ -186,8 +186,8 @@ router.post(
  *   }
  */
 router.get(
-  '/disputes/:disputeHash',
-  DisputeHandlers.getDisputeByHash
+  '/disputes/:arbitratorAddress/:disputeId',
```
Comment (n1c01a5) [inline routes/index.js]: idem `arbitrator/:arbitratorAddress/disputes/:disputeId`

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

File: routes/index.js (hunk 1 above)
Comment (satello) [reply routes/index.js]: good idea

**Review by n1c01a5 — APPROVED** [review]

**Pull request merged**